airflow/values.yaml

```diff
@@ -43,14 +40,16 @@ db:
     database: airflow
 
 persistence:
-  enabled: false
+  enabled: true
```

Comment:
set to false by default, user will enable it in their config.yaml

Reply:
agreed. that was a mistake